### DISCUSSION TITLE
Pets natalinos

### DISCUSSION THREAD

**Discussion by ghost** [leogianfagna/IssueTracker#996] [Ideia]
### Servidor afetado

Henesys

### Qual o propósito da ideia?

Inserir cosméticos e mecânicas que despertam o interesse do público (mudar a skin do pet e voar nas montarias)

## Com a temática natalina chegando gostaria de sugerir esses dois pets

# Mini Jogador:
- ataca player
- ataca mobs
- Armazena itens
- Você pode trocar a skin do Mini Jogador
(https://mcmodels.net/products/12695/nogs-mini-you-christmas-pets-plushies)

# Rena Natal:
- ataca mobs
- armazena itens
- Montaria que Voa

(https://mcmodels.net/products/12976/christmas-raindeer-pet)

- Se possível, seria ótimo acrescentar a possibilidade de escrever o nome do pet colorido.

### Aplicação

Mini Jogador:
https://mcmodels.net/products/12695/nogs-mini-you-christmas-pets-plushies

Rena Natal:
https://mcmodels.net/products/12976/christmas-raindeer-pet

### Referências

![image](https://github.com/user-attachments/assets/b34e7ac3-2cd2-4bfe-bb1e-abed0a76c48b)
![image](https://github.com/user-attachments/assets/502df005-12d4-4ccf-9508-69235f5241d8)

**Comment by leogianfagna** [comment]
@Yokiro27 você acha que o mini jogador seria interessante e faz relação com o tema do servidor?

**Reply by ghost** [reply]
Sim, Henesys tem um grande foco em personalização e cosméticos, então o mini jogador seria um pet bem divertido. Além disso, com a época atual, ele combina perfeitamente com o tema natalino.
